Enable MobileGestalt on Mac

megangardner opened the pull request:
#### 1f64f4960ee9b6a1cd0470219081b9c2aca8c35b
<pre>
Enable MobileGestalt on Mac
<a href="https://bugs.webkit.org/show_bug.cgi?id=253676">https://bugs.webkit.org/show_bug.cgi?id=253676</a>
rdar://106523424

Reviewed by NOBODY (OOPS!).

* Source/WebCore/PAL/pal/spi/ios/MobileGestaltSPI.h:
</pre><!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1f64f4960ee9b6a1cd0470219081b9c2aca8c35b

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/111975 "2 style errors") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/21103 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/90/builds/588 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/3739 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/120637 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/116035 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/22452 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/12251 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/86/builds/3485 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/117735 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/16670 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/99830 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/104996 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/98622 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/88/builds/390 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/45654 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/13533 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/89/builds/408 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/93047 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/14208 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/9816 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/19582 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/52412 "Passed tests") | | 
| [❌ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/8038 "") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/16015 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

pxlcoder commented on Source/WebCore/PAL/pal/spi/ios/MobileGestaltSPI.h:
```diff
@@ -27,8 +27,6 @@
 
 #import <wtf/Platform.h>
 
-#if PLATFORM(IOS_FAMILY)
```
Is `MobileGestalt.h` available on macOS? I wasn't able to find it.

megangardner replied:
Yes it is

pxlcoder replied:
(I found it now)

pxlcoder reviewed (commented):
We should move this header to `pal/spi/cocoa` if it's being added to macOS.

pxlcoder reviewed (dismissed):
Looks good after moving the header into the `cocoa` folder.

achristensen07 reviewed (commented):
Should we move this file from pal/spi/ios to pal/spi/cocoa?

megangardner committed:
Enable MobileGestalt on Mac
https://bugs.webkit.org/show_bug.cgi?id=253676
rdar://106523424

Reviewed by NOBODY (OOPS!).

* Source/WebCore/PAL/pal/spi/ios/MobileGestaltSPI.h:
